File: lab1/app/src/main/java/chernook/fit/bstu/by/MainActivity.java
package chernook.fit.bstu.by;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;
import android.widget.TextView;

import chernook.fit.bstu.by.text.TextFunction;

public class MainActivity extends AppCompatActivity {
    //TODO add a new method
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        for (int count = 0; count < 10; count++) {
            Log.d("MainActivity", " counter =" + count);
        }
        displayText();

    }

    private void displayText() {
        TextFunction tf = new TextFunction();
        TextView nt = findViewById(R.id.newtest);
        nt.setText(tf.getValue());
    }
}

File: lab12/app/src/main/java/chernook/fit/lab12/MainActivity.kt
package chernook.fit.lab12

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import kotlinx.coroutines.Dispatchers
import kotlinx.coroutines.GlobalScope
import kotlinx.coroutines.launch

class MainActivity : AppCompatActivity() {
    private var buttonVk: Button?=null
    private var buttonWeather: Button?=null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        buttonVk = findViewById(R.id.buttonVkActivity)
        buttonWeather = findViewById(R.id.buttonWeatherActivity)

        buttonVk!!.setOnClickListener {
            val intent = Intent(this, VKActivity::class.java)
            startActivity(intent)
        }
        buttonWeather!!.setOnClickListener {
            val intent = Intent(this, WeatherActivity::class.java)
            startActivity(intent)
        }
    }
}